src/shape/bounding-box.js

```diff
@@ -421,6 +421,12 @@ class BoundingBox {
         return sq;
     }
 
+    area() {
+        const extents = this.halfExtents.clone();
+        extents.mulScalar(2);
+        return extents.x * extents.y + extents.z * extents.x + extents.y * extents.z;
+    }
+
```

Comment:
this does not seem to be used in this PR .. can it be deleted?

Reply:
It's used in the `findBestSplitPlane` function to evaluate surface area sizes when considering the best bin to split along; should I move the function to the bvh class since it's only used there?